### ISSUE TITLE
Repeated fields have only one field (with last value) in bag-info.txt

### ISSUE THREAD

**Issue by bonniegee** [APTrust/dart-runner#7]
I created a job with the following tags from the job JSON:
```
'tags': [{'tagFile': 'bag-info.txt', 'tagName': 'ArchivesSpace-URI', 'value': '/repositories/2/archival_objects/1125043'}, {'tagFile': 'bag-info.txt', 'tagName': 'Start-Date', 'value': '1945-01-01'}, {'tagFile': 'bag-info.txt', 'tagName': 'End-Date', 'value': '1945-12-31'}, {'tagFile': 'bag-info.txt', 'tagName': 'Origin', 'value': 'digitization'}, {'tagFile': 'bag-info.txt', 'tagName': 'Rights-ID', 'value': '1'}, {'tagFile': 'bag-info.txt', 'tagName': 'Rights-ID', 'value': '6'}]
```

In this example, I am expecting the bag-info.txt file to have `Rights-ID` repeated twice (once with a value of `1`, the other with a value of `6`). However, it only appears once in the bag-info.txt file:

```
Bag-Size: 3.9 GB
Bagging-Date: 2022-06-09T14:56:49Z
Bagging-Software: DART Runner v0.93-beta for Darwin x86_64 (Build 9556279 2022-06-07)
Payload-Oxum: 4174507844.136
ArchivesSpace-URI: /repositories/2/archival_objects/1125043
Start-Date: 1945-01-01
End-Date: 1945-12-31
Origin: digitization
Rights-ID: 6
BagIt-Profile-Identifier: zorya_bagit_profile.json
```

**Comment by diamondap** [comment]
Sorry I didn't see this earlier. Just tested, and I can reproduce the problem locally. I'll let you know when we have a fix.

**Comment by bonniegee** [comment]
Thank you, @diamondap!

**Comment by diamondap** [comment]
This is fixed in version v0.94-beta. Links to the latest downloads are at https://aptrust.github.io/dart-docs/users/dart_runner/#downloads

**Comment by bonniegee** [comment]
Terrific, thank you for this fix (just tagging in @helrond)!
